internal/utils/templates/initial_schemas/14.sql

```diff
@@ -2314,6 +2325,16 @@ GRANT ALL ON FUNCTION graphql.variable_definitions_sort(variable_definitions jso
 GRANT ALL ON FUNCTION graphql.variable_definitions_sort(variable_definitions jsonb) TO service_role;
 
 
+-- --
+-- -- Name: FUNCTION graphql("operationName" text, query text, variables jsonb, extensions jsonb); Type: ACL; Schema: graphql_public; Owner: supabase_admin
+-- --
+-- 
+-- GRANT ALL ON FUNCTION graphql_public.graphql("operationName" text, query text, variables jsonb, extensions jsonb) TO postgres;
+-- GRANT ALL ON FUNCTION graphql_public.graphql("operationName" text, query text, variables jsonb, extensions jsonb) TO anon;
+-- GRANT ALL ON FUNCTION graphql_public.graphql("operationName" text, query text, variables jsonb, extensions jsonb) TO authenticated;
+-- GRANT ALL ON FUNCTION graphql_public.graphql("operationName" text, query text, variables jsonb, extensions jsonb) TO service_role;
```

Comment:
Shall we remove these? I will look into ways to not let pg_dump emit these statements altogether.

Reply:
I think it's more helpful to just show it but comment it out, so people can see what the raw pg_dump output looks like. Tuning pg_dump to omit these would be ideal though